cleanup: remove useCanvasTransformSync composables.

DrJKL opened the pull request:
No I am not proud of the new placeholder arguments.

## Summary

Small change to unify the two composables before updating the logic.
Edit: Now unifies them both into the **void**.

┆Issue is synchronized with this [Notion page](https://www.notion.so/PR-5742-cleanup-unify-useCanvasTransformSync-composables-2776d73d36508147ad39d11de8588b2e) by [Unito](https://www.unito.io)

DrJKL committed:
cleanup: unify useCanvasTransformSync composables.
No I am not proud of the new placeholder arguments.

jtydhr88 reviewed (dismissed):
LGTM

Myestery reviewed (dismissed):
Explicitly passing out undefined looks weird too
Can we use an object instead as parameter?

DrJKL commented:
> Does this failure indicate a change in the behavior https://19cb4803.comfyui-playwright-chromium.pages.dev/#?testId=0bebc894f304375862db-e6498858489872433645?

Very probably

DrJKL committed:
fix: Get the canvas from the store externally for the users of the original composable.

DrJKL commented:
> Does this failure indicate a change in the behavior https://19cb4803.comfyui-playwright-chromium.pages.dev/#?testId=0bebc894f304375862db-e6498858489872433645?

Okay, let's see if that worked.

DrJKL committed:
Cleanup: remove useCanvasTransformSync (#5748)

## Summary

Replace the bespoke composable with a vueuse one that does the same
thing but a little nicer.

┆Issue is synchronized with this [Notion
page](https://www.notion.so/PR-5748-Cleanup-remove-useCanvasTransformSync-2786d73d365081c29c56f4133af9bf80)
by [Unito](https://www.unito.io)

DrJKL commented:
Just going to combine this with the subsequent cleanup PR.

Myestery reviewed (approved):
LGTM
